### PR TITLE
Fixups for python3 and noetic

### PR DESCRIPTION
**Pull request by lucbettaieb**
Fixups for compatibility with ROS Noetic + python3

Companion PR for `franka_ros_interface`
https://github.com/justagist/franka_ros_interface/pull/30

### CODE REVIEW

**Comment by lucbettaieb** [comment]
@justagist I'm not a super pythonic individual, so if there's a better way of doing the imports over `from .thing import *` please lemme know and I'll revise the PR.

Hope this is helpful!

**Comment by lucbettaieb** [comment]
I haven't tested this a bunch yet, but will be doing so for one of my own projects.  However, I've been able to launch it without a bunch of errors popping up in the launching terminal.

Here's a screenscast:
https://drive.google.com/file/d/128UMZ7vXNV5PtdDFBiyChlcVvUT9vY42/view?usp=sharing

**Comment by lucbettaieb** [comment]
Also, I see you may not be a fan of my changes to the requirements.txt file per another PR.  Not having warnings/errors when launching is pretty nice, but its your project and not required for it to run, so lemme know and I'll remove them.

**Comment by justagist** [comment]
Hi, thanks for this PR as well!
The complication with porting the simulator to Noetic was getting Moveit to work (the other changes are trivial) because panda_moveit_config had not been fully integrated with Noetic when I had tested. I don't know if they have fixed this now; have you checked if this works correctly with your modifications?

Obviously, the main reason for writing this simulator was to be able to use the same interfaces and code interchangeably with the real robot, so I can review this only after I am done reviewing your [other PR](https://github.com/justagist/franka_ros_interface/pull/30) (There are also a couple of other major changes planned to the simulator, which may slow the review of this PR even further unfortunately). 

> Also, I see you may not be a fan of my changes to the requirements.txt file per another PR. Not having warnings/errors when launching is pretty nice, but its your project and not required for it to run, so lemme know and I'll remove them.

Haha, yes, sorry! I would rather not have unnecessary packages added as dependencies to this repository just to remove insignificant warning messages. But this is not a big problem, and can be dealt with later. The main things that concern me are the following, and your help will be greatly appreciated:
1. Moveit compatibility
2. Franka ROS Interface compatibility (easiest way might be to just test if all the demos mentioned in the README of this repo run correctly)

Thanks again for this PR, and very sorry that I am unable to review it soon. In the meantime, I will direct anyone who is looking for a Noetic version of this simulator to your fork so that my delay doesn't affect them.

-- Saif

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,8 @@
 
 # Linux
 *.swp
+
+# ROS Stuff
+.rosinstall
```
Comment (justagist) [inline .gitignore:35]: Why have you added the .rosinstall file to gitignore?

File: panda_sim_custom_action_server/src/panda_sim_custom_action_server/bezier.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```
Comment (justagist) [inline panda_sim_custom_action_server/src/panda_sim_custom_action_server/bezier.py:1]: Any specific reason for adding the shebang?

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,4 @@
 numpy 
-numpy-quaternion==2020.5.11.13.33.35
\ No newline at end of file
+numpy-quaternion==2020.5.11.13.33.35
+scipy
+numba
```
Comment (justagist) [inline requirements.txt:4]: These packages are not required for building or running the simulator, and do not add any improvement.

**Comment by justagist** [comment]
Hi, I started working on a separate branch entirely (now in testing phase) for ROS Noetic + Python3 support because significant changes were required for other files not included in your PR, and most of the changes in this PR were not needed. So it was easier to just start from scratch. 
I am closing this PR, but please feel free to raise another one if you want to add more changes. The current branch (beta phase) relevant to your PR is [_noetic_test](https://github.com/justagist/panda_simulator/tree/_noetic_test).

**Comment by lucbettaieb** [comment]
@justagist Sweet!  No worries, glad I was able to at least help you get started.

**Pull request closed without merging**